data/json/items/gunmod/barrel.json

```diff
@@ -72,5 +72,39 @@
     "dispersion_modifier": 500,
     "ups_charges": 50,
     "min_skills": [ [ "gun", 1 ] ]
+  },
+  {
+    "id": "lead_barrel_big",
+    "copy-from": "barrel_big",
+    "type": "GUNMOD",
+    "name": "Leadworks barrel extension",
+    "description": "A Leadworks built in longer barrel increases the muzzle velocity of a firearm, contributing to both accuracy and damage slows aiming.",
+    "price": 0,
+    "damage_modifier": 3,
+    "range": 3,
+    "flags": [ "IRREMOVABLE" ]
+  },
+  {
+    "id": "lead_barrel_small",
+    "copy-from": "barrel_small",
+    "type": "GUNMOD",
+    "name": "Leadworks shortened barrel",
+    "description": "A Leadworks built in shortened barrel results in reduced accuracy, and a minor increase in noise, but also greatly improves the ease with which the weapon can be carried and wielded.",
+    "price": 0,
+    "dispersion_modifier": 60,
+    "range": -6,
+    "flags": [ "IRREMOVABLE" ]
+  },
+  {
+    "id": "lead_barrel_heavy_duty",
+    "copy-from": "barrel_big",
+    "type": "GUNMOD",
+    "name": "Leadworks heavy duty barrel",
+    "description": "A Leadworks built in heavy duty barrel especially designed for prolonged shooting. Reduce accuracy but increase damage output and weapon range.",
+    "price": 0,
+    "damage_modifier": 3,
+    "dispersion_modifier": 60,
```

Comment:
What's the rationale for these numbers? How does it increase muzzle velocity and effective range while reducing accuracy?

Reply:
It is debuff for LMG. Originally L523-LMG has +60 dispersion. Now it comes from that mod.

Reply:
In fact it should be 90 bacues gyro stybilizer reduce dispersion to 30.

Reply:
+90 is even worse. Increasing muzzle velocity and effective range while decreasing accuracy has no grounding in reality. If the weapon's stats were originally way out they need fixing here.